iron-list.html

```diff
@@ -1390,11 +1390,14 @@
 
       forceUpdate = forceUpdate || this._scrollHeight === 0;
       forceUpdate = forceUpdate || this._scrollPosition >= this._estScrollHeight - this._physicalSize;
+      // Probably a bug that style.height is a string like '100px' and it is being compared to
+      // a number?
       forceUpdate = forceUpdate || this.grid && this.$.items.style.height < this._estScrollHeight;
 
       // Amortize height adjustment, so it won't trigger large repaints too often.
       if (forceUpdate || Math.abs(this._estScrollHeight - this._scrollHeight) >= this._optPhysicalSize) {
         this.$.items.style.height = this._estScrollHeight + 'px';
+        this.updateViewportBoundaries();
```

Comment:
Also, looking at the stack trace of the Settings page:

<img width="908" alt="screen shot 2016-11-01 at 10 09 12 am" src="https://cloud.githubusercontent.com/assets/1410613/19901770/7f18c63a-a025-11e6-89e7-0c8bd0e0bdcd.png">

It's calling `notifyResize()` from a `setInterval`.  Another way is to set fire `notifyResize` only when the animated page first renders (before the animation starts). However that would require that the scroll target's height is set. See: https://github.com/PolymerElements/iron-list#sizing-iron-list